exposee-ui/src/components/LoadingSpinner/LoadingSpinner.css

```diff
@@ -0,0 +1,40 @@
+.loading-spinner {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+  }
+
+  .loading-text {
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+
+  .dots-container {
+    display: flex;
+  }
+
+  .dot {
+    width: 10px;
+    height: 10px;
+    margin: 2px;
+    border-radius: 50%;
+    background-color: #3498db; /* Change the color of the dots if needed */
+    animation: bounce 0.5s infinite alternate;
+  }
+
+  .dot:nth-child(2) {
+    animation-delay: 0.15s;
+  }
+
+  .dot:nth-child(3) {
```

Comment:
haha this is a really impressive way to make the loading spinner!!! 
In most cases, it would make sense to just use a gif for spinner, and then in react show the spinner by default until the response comes back, then you show the actual content.